views/tasks.py

```diff
@@ -65,3 +66,9 @@ def notify_subscribers():
     notifier = logic.notifier.notifier_for_event(request.json['event'])(**request.json['options'])
     notifier.notify()
     return Response(status=200)
+
+
+@app.route('/tasks/lovelinks/cleanup', methods=['GET'])
```

Comment:
I remember reading somewhere in the documentation that only GET is supported. Maybe I'm wrong. To be honest, GET doesn't make too much sense.